app/src/main/java/com/concordia/mcga/lib/BuildingBottomSheetInfo.java

```diff
@@ -0,0 +1,1168 @@
+package com.concordia.mcga.lib;
```

Comment:
absolutely no idea what this file does. Also, it looks the same file as before minus the copyright information.